guides/source/routing.md

```diff
@@ -667,9 +704,9 @@ The `params` will also include any parameters from the query string. For example
 get 'photos/:id', to: 'photos#show'
 ```
 
-An incoming path of `/photos/1?user_id=2` will be dispatched to the `show` action of the `Photos` controller. `params` will be `{ controller: 'photos', action: 'show', id: '1', user_id: '2' }`.
+An incoming `GET` request for `/photos/1?user_id=2` will be dispatched to the `show` action of the `PhotosController` class as usual and the `params` hash will be `{ controller: 'photos', action: 'show', id: '1', user_id: '2' }`.
```

Comment:
I _think_ both this section and the next one are also applicable to resourceful routes. This is not a must-have for this PR, but do you have any ideas for a way to make it clear that you can use this in resourceful routes as well?

Reply:
I was thinking that, I'll make a follow-on task to review and test all of these features in this section. More than one are likely applicable to resourceful routes...and things can be restructured a bit to make that clear.